packages/kbn-server-http-tools/src/ssl/ssl_config.ts

```diff
@@ -161,6 +162,13 @@ export class SslConfig {
         : secureOptions | secureOption; // eslint-disable-line no-bitwise
     }, 0);
   }
+
+  public isEqualTo(otherConfig: SslConfig) {
+    if (this === otherConfig) {
+      return true;
+    }
+    return isEqual(this, otherConfig);
+  }
```

Comment:
Had to implement a comparison func on the `SslConfig` class. Wasn't strictly necessary, but it allows to know when the SSL config really changed to avoid propagating the changes on every SIGHUP (and also to have better logging given we only output an info message when the config really changed)